website/content/docs/k8s/l7-traffic/failover-tproxy.mdx

```diff
@@ -0,0 +1,124 @@
+--- 
+layout: docs
+page_title: Configure failover service instances
+description: Learn how to define failover services in Consul on Kubernetes when proxies are in transparent proxy mode. Consul can send traffic to backup service instances if a destination service becomes unhealthy or unresponsive. 
```

Comment:
I saw this shows up under Failover for mesh in general as well. I think it's fine for RC, but perhaps we can consider also describing the workflow for VMs as well, as just K8s is now explicitly documented? 

<img width="317" alt="image" src="https://github.com/hashicorp/consul/assets/551323/b38ec738-1bb9-4741-82ea-f82c1b52008f">